cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -181,9 +187,7 @@ download_kernel_src() {
   popd
 }
 
-install_cross_toolchain_pkg() {
-  mkdir -p /build
-  pushd /build
+get_cross_toolchain_pkg() {
```

Comment:
Can you please add some docstrings for this function? Especially why this is necessary even though we already have $TOOLCHAIN_ARCHIVE